string new_path_name
---
string old_path_name